scripts/install-binary.mjs

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+'use strict'
+
+import { spawn } from 'node:child_process'
+import {
+  chmodSync,
+  createWriteStream,
+  existsSync,
+  mkdirSync,
+  renameSync,
+} from 'node:fs'
+import { dirname, join } from 'node:path'
+import { Readable } from 'node:stream'
+import { fileURLToPath } from 'node:url'
+import { x } from 'tar'
```

Comment:
import { tar } from 'tar'

Reply:
It doesn't have a named `tar` export, but I changed it to `import { extract } from 'tar'` to be more clear